src/test/java/core/basesyntax/service/impl/ReaderServiceImplTest.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReaderService;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class ReaderServiceImplTest {
+    private static String filePath;
+    private static ReaderService reader;
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
```

Comment:
let's instead add an attribute `expected` to annotation Test, the same with others